lib/spack/spack/test/config.py

```diff
@@ -29,16 +29,22 @@
 # sample config data
 config_low = {
     'config': {
-        'install_tree': {'root': 'install_tree_path'},
+        'install_trees': {
+            'spack-root': {
+                'root': 'install_tree_path'}},
         'build_stage': ['path1', 'path2', 'path3']}}
 
 config_override_all = {
     'config:': {
-        'install_tree:': {'root': 'override_all'}}}
+        'install_trees': {
```

Comment:
By omitting the `:` at the end (e.g. using `'install_trees':` vs. `'install_trees:':`) I don't think this is testing the same thing. The extra `:` is intended to be used to override the entire dictionary (otherwise Spack will merge the two dictionaries). I'll have to think of how best to adapt these tests.

Reply:
Hmm ok. Would it be alright if I let you figure out exactly how to adapt these tests?

Reply:
Yes I'll figure out how to modify these